src/components/CertificateTemplates/tlds/sg/gov/seab/common/template.js

```diff
@@ -64,44 +64,32 @@ export const EXAM_LVL_TEXT = {
 
 export const FORMATDATEPREFIX = dateString => {
   const date = new Date(dateString);
-  const day = date.getDate();
-  const month = date.getMonth();
-  const year = date.getFullYear();
-
-  let dayValue = "";
-
-  const months = [
-    "01",
-    "02",
-    "03",
-    "04",
-    "05",
-    "06",
-    "07",
-    "08",
-    "09",
-    "10",
-    "11",
-    "12"
-  ];
-
-  const lengthday = day.toString().length;
-  if (lengthday === 1) {
-    dayValue = `0${day}`;
-  } else {
-    dayValue = day;
-  }
+  const day = date.toLocaleDateString("en-SG", {
+    timeZone: "Asia/Singapore",
+    day: "2-digit",
+  });
+  const month = date.toLocaleDateString("en-SG", {
+    timeZone: "Asia/Singapore",
+    month: "2-digit",
+  });
+  const year = date.toLocaleDateString("en-SG", {
+    timeZone: "Asia/Singapore",
+    year: "numeric",
+  });
```

Comment:
haha seems a little redundant though
```
console.log(new Date().toLocaleDateString('en-SG', {
    timeZone: "Asia/Singapore",
    day: "2-digit",
  	month: "2-digit",
  	year: "numeric",
  }));
  ```
see whether u want to edit, im fine either

Reply:
Haha oh ya, that's more elegant. Let me update it to this then 🤓